docs/docs/cmd/spo/site/site-sharingpermission-set.mdx

```diff
@@ -46,7 +46,7 @@ m365 spo site sharingpermission set --siteUrl https://siteaddress.com/sites/site
 Update the sharing permissions for a site where so owners can share the site, but members can only share files
 
 ```sh
-m365 spo site sharingpermission set --siteUrl https://siteaddress.com/sites/sitename  --capability full
+m365 spo site sharingpermission set --siteUrl https://siteaddress.com/sites/sitename  --capability limited
```

Comment:
```suggestion
m365 spo site sharingpermission set --siteUrl https://siteaddress.com/sites/sitename --capability limited
```

Could you fix the double space as well? 😄

Reply:
Auch!